homeassistant/components/modbus/binary_sensor.py

```diff
@@ -1,11 +1,12 @@
 """Support for Modbus Coil sensors."""
 import logging
+from typing import Optional
 
 import voluptuous as vol
 
 from homeassistant.components.binary_sensor import BinarySensorDevice
-from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.const import CONF_NAME, CONF_SLAVE
+from homeassistant.components.sensor import DEVICE_CLASSES_SCHEMA, PLATFORM_SCHEMA
```

Comment:
You cannot import the device class schema for sensor into binary sensor integration.

Reply:
We also shouldn't import PLATFORM_SCHEMA from sensor either.

Reply:
@balloob, my mistake, I will fix the ```DEVICE_CLASSES``` schema. However, the ```PLATFORM_SCHEMA``` was imported from sensor already before my change. Could I fix that also in this PR?

Reply:
Rebased against latest dev, resolved all conflicts. Please take a look at it.